cpp/box/Box.h

```diff
@@ -263,13 +263,18 @@ class Box
         return delta;
     }
 
-    void makeFractional(vec3<float>* vecs, unsigned int Nvecs) const
+    //! Convert point coordinates from absolute to relative box coordinates.
```

Comment:
I don't think "relative" and "fractional" mean the same thing. "Relative" suggests a translation (in both physics and CSS contexts), whereas fractional conveys the desired meaning to me.
```suggestion
    //! Convert point coordinates from absolute coordinates to fractional box coordinates.
```

Reply:
I agree.